libs/csm/rest/csm_rest_quota.py

```diff
@@ -151,15 +151,17 @@ def verify_max_size(self, max_size: int, akey: str, skey: str, bucket: str):
         obj_name=f'{obj_name_prefix}{time.perf_counter_ns()}'
         self.log.info("Perform Put operation for 1 object of max size")
         res = s3_misc.create_put_objects(obj_name, bucket,
-                       akey, skey, object_size=int(max_size/(1024*1024)))
+                                         akey, skey, object_size=int(max_size/1024),
+                                         block_size="1K")
         obj_list.append(obj_name)
         if res:
             obj_name=f'{obj_name_prefix}{time.perf_counter_ns()}'
             self.log.info("Perform Put operation of Random size and 1 object")
-            random_size = self.cryptogen.randrange(1, max_size)
+            random_size = self.cryptogen.randrange(1, 128)
```

Comment:
We should use self.random_gen.randrange(1,
instead of cryptogen

Reply:
done